src/base/bittorrent/session.cpp

```diff
@@ -1117,13 +1120,16 @@ void Session::prepareStartup()
     connect(this, &Session::torrentsLoaded, context, [this, context](const QVector<Torrent *> &torrents)
     {
         context->processingResumeDataCount -= torrents.count();
+        context->finishedResumeDataCount += torrents.count();
         if (!context->isLoadedResumeDataHandlingEnqueued)
         {
             QMetaObject::invokeMethod(this, [this, context]() { handleLoadedResumeData(context); }, Qt::QueuedConnection);
             context->isLoadedResumeDataHandlingEnqueued = true;
         }
         m_nativeSession->post_torrent_updates();
         m_refreshEnqueued = true;
+
+        emit startupProgressUpdated((context->finishedResumeDataCount * 100.) / context->totalResumeDataCount);
```

Comment:
Maybe use `float` instead of `double`:
```suggestion
        emit startupProgressUpdated((context->finishedResumeDataCount * 100.f) / context->totalResumeDataCount);
```

Reply:
> Maybe use `float` instead of `double`:

What is the reason not to use the default type, provided that you don't care about the insignificant performance tuning?

Reply:
ok, forget it.